Update bullet chart tutorial (#214)
patternfly/training-scenarios: react-charts/bullet-chart/step2.md

Create a simple bullet chart.

1. <strong>Navigate to the `src` folder and open `src/App.js`{{open}}</strong>

2. <strong>Set up the general structure of the chart and the necessary import statements.</strong>

> Copy the following code into the App.js file, replacing all of the content there:

Reviewer: This needs tics around `App.js`.

![Screen Shot 2019-12-09 at 10 37 52 AM](https://user-images.githubusercontent.com/17481322/70449494-163d5900-1a70-11ea-8a5b-2fc5e70120aa.png)

> Copy the following code into the App.js file, replacing all of the content there:

Author: ```suggestion
Copy the following code into the `App.js` file, replacing all of the content there:
```

<pre class="file" data-filename="App.js" data-target="replace">
import React from 'react';
import "@patternfly/react-core/dist/styles/base.css";
import './app.css';
import {
  Chart,
  ChartBullet,
  ChartThemeColor
} from '@patternfly/react-charts';

function App() {
  return (
    &lt;div className=&quot;chart-container&quot;&gt;
      &lt;ChartBullet

>         primarySegmentedMeasureData={[{ name: &#39;Measure&#39;, y: 60 }]}

Reviewer 2: Ideally, all of these props (or atleast the main ones) are split up into individual steps and you explain what each prop does at each step

Author: Resolved

        qualitativeRangeData={[{ name: &#39;Range&#39;, y: 50 }, { name: &#39;Range&#39;, y: 75 }]}
        comparativeWarningMeasureData={[{ name: &#39;Warning&#39;, y: 88 }]}
        maxDomain={{y: 100}}
        height={150}
        width={600}
      /&gt;
    &lt;/div&gt;
  );
}

export default App;
</pre>

> - The `maxDomain` property defines the maximum domain value for a chart. In other words, the maximum value that can be expressed on each axis.

Reviewer 2: We haven't been using bullet points here, but I don't really think its too much of a problem, what do you think?

- The `primarySegmentedMeasureData` property adds a primary measure to the bullet chart. These values are the primary data points and appear as a blue bar by default.
- The `qualitativeRangeData` property adds qualitative ranges to the bullet chart. These values are used to express qualitative ranges like 'good' or 'bad' and appear on the chart as a larger grayscale box in the background by default.
- The `comparativeWarningMeasureData` property adds comparative warning measures to the bullet chart. These values are used to express a warning threshold and appear on the chart as a yellow tick mark.

> When the server reloads, you should see something like this:

Reviewer 3: ```suggestion
When the preview reloads, it  should look like something like this:
```

Author: ```suggestion
When the preview reloads, it should look something like this:
```
